swagger.yaml

```diff
@@ -1093,6 +1094,264 @@ paths:
+                  type: string
+                  description: The comment text.
+      responses:
+        "200":
+          description: Comment created successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardId:
+                    type: string
+                    description: The ID of the Trello card.
+                  comment:
+                    type: string
+                    description: The comment text.
+        "400":
+          description: Missing required parameters.
+        "500":
+          description: Failed to create comment.
+
+  "/card":
+    post:
+      operationId: createCard
+      summary: Create a new Trello card
+      description: Adds a new card to a Trello board.
+      tags:
+        - Trello
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                title:
+                  type: string
+                  description: The title of the Trello card.
+                description:
+                  type: string
+                  description: Description of the Trello card.
+      responses:
+        "200":
+          description: Card created successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardId:
+                    type: string
+                  title:
+                    type: string
+                  description:
+                    type: string
+        "400":
+          description: Missing required parameters.
+        "500":
+          description: Failed to create card.
+
+    get:
+      operationId: listCards
+      summary: Get all Trello cards on a list
+      description: Retrieves a list of cards and their comments from a specified Trello list.
+      tags:
+        - Trello
+      responses:
+        "200":
+          description: List of cards retrieved successfully.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Card"
+                  
+        "500":
+          description: Failed to fetch cards.
+
+  "/board-members":
+    get:
+      operationId: getBoardMembersEmails
+      summary: Get Trello board members' emails
+      description: Retrieves the email addresses of Trello board members.
+      tags:
+        - Trello
+      responses:
+        "200":
+          description: List of board members' emails retrieved successfully.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Member"
+        "500":
+          description: Failed to fetch board members' emails.
+
+  "/card/{id}":
```

Comment:
Typically we would want the card endpoints to be grouped together, so this card/{id} should come straight after / card